acai-ios/SectionControllers/HeaderListSectionController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  HeaderListSectionController.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/18/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+
+class HeaderListSectionController: ListSectionController {
+    
+    var title: String!
+    
+    override init() {
```

Comment:
you can delete this, `init()` calls the `super` implicitly

Reply:
+1